docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM  xychelsea/ffmpeg-nvidia:latest-jupyter
+
+RUN conda install anaconda-client -n base
+RUN conda update conda
+
+RUN conda install python=3.9
+RUN conda install pip
+RUN conda install scikit-learn
+RUN conda install tensorboard
+RUN conda install pytest
+RUN conda install pingouin
+RUN conda install -c conda-forge ffmpeg
+
+RUN pip install icatcher
```

Comment:
1) torch is not being installed prior to icatcher, which means the container will not have GPU support (as the default torch installation is CPU-only).
2) I believe there is no great way to ensure a GPU-installed torch will work, since they tend to frequently change the way you install it. maybe a prebuilt wheel using pip?, in any case, I don't see a way around someone actively making sure a prebuilt wheel works, requiring an active maintainer for this container (who will that be?)